### PR TITLE
feat: create sms_send in comunication_services

### PR DESCRIPTION
**Pull request by luizpcardoso**
add the libraty request in requirements.txt
the comunication_services.py contains the functions used to communicate with the user

### CODE REVIEW

File: app/services/comunication_services.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from os import getenv
+import requests
+import json
```
Comment (penguinuux) [inline app/services/comunication_services.py]: Remove unused imports.

File: .env.example [hunk 2]
```diff
@@ -1,3 +1,4 @@
 FLASK_ENV=development
 DATABASE_URI=postgresql://YOUR_USER:YOUR_PASSWORD@localhost:5432/YOUR_DATABASE
 SECRET=
+SMS_URI_KEY=
```
Comment (penguinuux) [inline .env.example]: Add a new line in the end of file.

File: app/services/comunication_services.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    SMS_URI = 'https://api.smsdev.com.br/v1/send'
+
+    data={
+    "key" : getenv("SMS_URI_KEY"),
+    "type" : 9,
+    "number" : phone,
+    "msg" : text
+    }
+    req = requests.post(SMS_URI, data=data)
+
```
Comment (penguinuux) [inline app/services/comunication_services.py]: Add a new line in the end of file.

**Review by penguinuux — CHANGES_REQUESTED** [review]

**Review by SidnySilva — APPROVED** [review]

File: .env.example (hunk 2 above)
Comment (marcelosj3) [reply .env.example]: Probably this is due because of the lack of using `black .`

**Review by marcelosj3 — APPROVED** [review]

File: .env.example (hunk 2 above)
Comment (luizpcardoso) [reply .env.example]: Done.

File: app/services/comunication_services.py (hunk 3 above)
Comment (luizpcardoso) [reply app/services/comunication_services.py]: Done.

**Review by marcelosj3 — APPROVED** [review]

**Review by penguinuux — APPROVED** [review]

**Pull request merged**